test_runner/src/main/kotlin/ftl/util/ObfuscationGson.kt

```diff
@@ -19,7 +19,7 @@ val obfuscatePrettyPrinter = GsonBuilder()
 internal object ListOfStringTypeToken : TypeToken<List<String>>()
 
 @VisibleForTesting
-internal object ListOfStringListTypeToken : TypeToken<ShardChunks>()
+internal object ListOfStringListTypeToken : TypeToken<List<Map<String, List<String>>>>()
```

Comment:
This change from ShardChunks to List<Map<...>> looks crazy. Is it worth not having another alias here like before?
maybe `CustomShardChunks`?
WDYT? Worth it?

Reply:
Why not! 👍🏻 
Personally, I don't like type aliases used as cover for data structures but I think I'm in the minority